src/FunctionStateMap.js

```diff
@@ -0,0 +1,24 @@
+export default {
+
+  // Accept Ended or Paused.
+  pauseVideo: {
+    acceptableStates: [0, 2],
```

Comment:
It is not clear when scanning the code what are the 0, 1 and 2s. Please use constants for this.